src/App.tsx

```diff
@@ -1,24 +1,23 @@
-import { useState } from 'react'
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import { AuthContextProvider } from "./contexts/AuthContext";
```

Comment:
adding auth context was not part of this ticket and was not discussed thoroughly in order to be implemented. this should be removed

Reply:
removed auth context